frontend/packages/noobaa-storage-plugin/integration-tests/tests/bucketClassTest.scenario.ts

```diff
@@ -0,0 +1,98 @@
+    await ocsOperator.click();
+    await browser.wait(until.visibilityOf(createBucketClassLink));
+    await click(createBucketClassLink);
+  });
+
+  afterEach(async () => {
+    await crudView.clickDetailsPageAction('Delete Bucket Class');
+    await browser.wait(until.presenceOf($('#confirm-action')));
+    await $('#confirm-action').click();
+    await browser.sleep(100);
+  });
+
+  it('Create a 1 Tier(Spread) bucket class', async () => {
+    bcHandler.setTiers([Tier.SPREAD]);
+    const bc = await bcHandler.createBC();
+    expect(bc.name).toEqual(bcName);
+    expect(bc.namespace).toContain(NS);
+    expect(bc.tier1Policy.toLocaleUpperCase()).toContain(Tier.SPREAD);
+    const cliData = JSON.parse(
+      execSync(`kubectl get bucketclass ${bcName} -n openshift-storage -o json`).toString(),
+    );
+    expect(cliData?.metadata?.name).toEqual(bc.name);
+  });
+
+  it('Create a 1 Tier(Mirror) bucket class', async () => {
+    bcHandler.setTiers([Tier.MIRROR]);
+    const bc = await bcHandler.createBC();
+    expect(bc.name).toEqual(bcName);
+    expect(bc.namespace).toContain(NS);
+    expect(bc.tier1Policy.toLocaleUpperCase()).toContain(Tier.MIRROR);
+    const cliData = JSON.parse(
+      execSync(`kubectl get bucketclass ${bcName} -n openshift-storage -o json`).toString(),
+    );
+    expect(cliData?.metadata?.name).toEqual(bc.name);
+  });
+
+  it('Create a 2 Tier(Spread, Spread) bucket class', async () => {
+    bcHandler.setTiers([Tier.SPREAD, Tier.SPREAD]);
+    const bc = await bcHandler.createBC();
+    expect(bc.name).toEqual(bcName);
+    expect(bc.namespace).toContain(NS);
+    expect(bc.tier1Policy.toLocaleUpperCase()).toContain(Tier.SPREAD);
+    expect(bc.tier2Policy.toLocaleUpperCase()).toContain(Tier.SPREAD);
+    const cliData = JSON.parse(
+      execSync(`kubectl get bucketclass ${bcName} -n openshift-storage -o json`).toString(),
+    );
+    expect(cliData?.metadata?.name).toEqual(bcName);
+  });
+
+  it('Create a 2 Tier(Spread, Mirror) bucket class', async () => {
+    bcHandler.setTiers([Tier.SPREAD, Tier.MIRROR]);
+    const bc = await bcHandler.createBC();
+    expect(bc.name).toEqual(bcName);
+    expect(bc.namespace).toContain(NS);
+    expect(bc.tier1Policy.toLocaleUpperCase()).toContain(Tier.SPREAD);
+    expect(bc.tier2Policy.toLocaleUpperCase()).toContain(Tier.MIRROR);
+    const cliData = JSON.parse(
+      execSync(`kubectl get bucketclass ${bcName} -n openshift-storage -o json`).toString(),
+    );
+    expect(cliData?.metadata?.name).toEqual(bcName);
+  });
+
+  it('Create a 2 Tier(Mirror, Mirror) bucket class', async () => {
+    bcHandler.setTiers([Tier.MIRROR, Tier.MIRROR]);
+    const bc = await bcHandler.createBC();
+    expect(bc.name).toEqual(bcName);
+    expect(bc.namespace).toContain(NS);
+    expect(bc.tier1Policy.toLocaleUpperCase()).toContain(Tier.MIRROR);
+    expect(bc.tier2Policy.toLocaleUpperCase()).toContain(Tier.MIRROR);
+    const cliData = JSON.parse(
```

Comment:
You can create a common function for this, since its used in every test case here.

Reply:
I don't think this deserves to be a function.

Reply:
On a second thought seems like something we should do. will add it.